src/components/EditCoursePage/AdditionalMetadataFields.jsx

```diff
@@ -147,6 +154,16 @@ const AdditionalMetadataFields = (props) => {
           disabled={disabled}
           required={isRequiredField('start_date')}
         />
+        <Field
+          name="additional_metadata.end_date"
+          type="date"
+          component={DateTimeField}
+          dateLabel="End Date"
+          timeLabel={`End Time (${utcTimeZone})`}
+          utcTimeZone
+          disabled={disabled}
+          required={isRequiredField('end_date')}
+        />
```

Comment:
Note: Double check additional metadata required fields based on source in edx-internal, to make sure the change does not break AM flow for different sources who do not require the new fields.

Reply:
I have checked the `edx-internal` AdditionalMetadata config, and it only checks for the required fields in AdditionalMetadata. However, I submitted a pull request to add end_date to the list of required fields in AdditionalMetadata